docs/AboutUs.md

```diff
@@ -5,6 +5,6 @@ Display | Name | Github Profile | Portfolio
 ![](https://via.placeholder.com/100.png?text=Photo) | Daniel | [Github](https://github.com/daniellimws) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | Vishesh | [Github](https://github.com/wish2023) | [Portfolio](docs/team/johndoe.md)
 ![](https://via.placeholder.com/100.png?text=Photo) | Amanda | [Github](https://github.com/amanda-chua) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Alston | [Github](https://github.com/alstontham) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Alston | [Github](https://github.com/alstontham) | [Portfolio](docs/team/alston-tham.md)
```

Comment:
Try using `docs/team/alstontham.md` instead of `team/alstontham.md`
Change the file name to your github username
![image](https://user-images.githubusercontent.com/60333397/98456984-c013f100-21bd-11eb-913f-3e3fcdc70955.png)